### ISSUE TITLE
Errors are kicking me out of conferences

### ISSUE THREAD

**Issue by kriztan** [siacs/Conversations#1134]
I have noticed that me and some others were kicked out of conferences if the connection is very unstable, e.g. connected with EDGE on the lowest signal range with some short offline times. I've noticed too, that this happens only to privat conferences until now.
Can anybody confirm that?

**Comment by kriztan** [comment]
I've now updated to Conversations 1.6.0 beta and this bug is furthermore existing. During my last post in April I have to expand this issue to non private conferences too and it doesn't really happen only at bad/unstable network connections. 
This is a really annoying bug that should be fixed.

On the serverside, there is the latest prosody trunk running on centos. I don't know if this only a client problem or a server problem. 
Perhaps we can collect some more infos to solve that problem.

**Comment by kriztan** [comment]
This bug is really annoying. Since version 1.6.0 the kicks became more often, almost once a day for me and other users running Conversations > 1.6.0 on my server.
Am I the only one who is affected by this bug?

**Comment by philipflohr** [comment]
Does your server support XMPP stream management (XEP-0198 http://xmpp.org/extensions/xep-0198.html) ?
If not you can enable it using the  mod_smacks module.

**Comment by kriztan** [comment]
@philipflohr yes, it does. My server has all XEP supported by Conversations activated.

**Comment by iNPUTmice** [comment]
That's actually a bug in prosody that has just recently been fixed. I haven't gotten around updating my own server though.

**Comment by Zash** [comment]
@iNPUTmice Which Prosody bug is that exactly?

**Comment by Zash** [comment]
For closure and future readers who stumble upon this issue:

The underlying issue is described here:
https://prosody.im/issues/issue/648
TL;DR: Bug in routing of groupchat messages to sessions currently hibernating (disconnected but kept alive by mod_smacks).

This is fixed in Prosody 0.10 and trunk with mod_smacks: https://prosody.im/issues/issue/523#comment-8

Prosody 0.9 with libevent
